src/handlers/clickAnswerHandler.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+import selectAnswer from '../handlers/selectAnswers.js';
+import countCorrectAnswers from '../handlers/countCurrentAnswers.js';
+
+function clickAnswerHandler(event) {
+  const li = event.target;
+  selectAnswer(li.dataset.answerValue);
+  const countSpan = document.getElementById('counter-span');
+  console.log(countSpan);
+  countSpan.textContent = countCorrectAnswers();
+  return countSpan;
```

Comment:
Do you need to return here? It's an event handler so the return value shouldn't be used anywhere, right?